LICENSE

```diff
@@ -0,0 +1,18 @@
+Copyright 2019 EmTransit B.V.
```

Comment:
interesting legal name: `EmTransit B.V.` is this how we are registered in chamber of commerce?

Reply:
Yes.